cds-snc/cppd-medical-report-sails
# Add page-specific js for Condition form

## What this does

Brings back show/hide javascript for "other" fields on Conditions form(s)

## To test

- Fill out the Personal form
- navigate to `/en/conditions`
- Trigger the hidden fields by selecting `Unknown` in the two radio lists
- The field hides again when something else is selected
- Save the Condition and then Edit the condition to confirm same functionality on Edit
